app/app/src/main/java/me/bickositieff/raspio/ui/songSelect/SongSelectFragment.kt

```diff
@@ -0,0 +1,36 @@
+package me.bickositieff.raspio.ui.songSelect
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.RecyclerView
+import me.bickositieff.raspio.R
+
+class SongSelectFragment : Fragment() {
+
+
+    private val viewModel: SongSelectViewModel by viewModels()
+    private lateinit var mAdapter: SongSelectAdapter
```

Comment:
There's no reason for this to be an instance variable. Content is updated via LiveData